data/help.txt

```diff
@@ -128,3 +128,7 @@ help "trading"
 
 help "friendly disabled"
 	`This ship is disabled and needs your help to patch them up! Pressing <Board selected ship> will fly you to their ship and get them up and running again. Just be sure to do it when no one is trying to kill you, or they could get caught in the crossfire!`
+
+help "tributing"
+	`You fire a warning shot, and the fourth wall shudders and cracks. An automated message appears on your computer.`
+	`"Captain! Demanding tribute from planets is not integrated with the storyline! You will receive a daily income and gain unrestricted landing access to this location, at the cost of being considered an outlaw by their government, but any campaign missions will not take this into account, which may result in story events that don't make sense. Are you sure you want to proceed?"`
```

Comment:
```suggestion
	`"Captain! Demanding tribute from planets is not integrated with the storyline! You will receive a daily income and gain unrestricted landing access to this location, at the cost of being considered an outlaw by their government. Campaign missions will not take this into account, which may result in story events that don't make sense. Are you sure you want to proceed?"`
```
I think that sentence was way too long.

Reply:
I think it does need to have a "but" or "however" or similar, though. The point is to say, "you will get these gameplay effects, _but_ you could also get these immersion-breaking effects."

It could still be split into two sentences, starting the second with "However,"